src/Parser/Warlock/Affliction/Modules/Features/Abilities.js

```diff
@@ -135,6 +135,12 @@ class Abilities extends CoreAbilities {
         spell: SPELLS.UNENDING_RESOLVE,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         cooldown: 180,
+        castEfficiency: {
```

Comment:
you could add `importance: ISSUE_IMPORTANCE.MINOR` to castEfficiency if you only want defensive and utility spells to be minor issues. Then you could adjust the average and major thresholds specifically for the checklist and avoid the dirty fix.

Reply:
I tend to agree that time on CD isn't ideal, but max cast suffer similar issues. For instance if the encounter is 3 min. and 1 sec long and your cooldown is 3 min. max casts will be twice what it actually should be. Maybe the solution is to use something like cooldownDuration/encounterLength for cooldowns that are not supposed to be used on cooldown? Anyway we should probably discuss that on discord or in an issue instead of in this PR.

If you want to keep the dirty fix that's okay with me, although I'd prefer if you didn't since neither solution is ideal and the recommended thresholds are pretty forgiving anyway.